beacon_node/lighthouse_network/gossipsub/src/types.rs

```diff
@@ -123,8 +123,10 @@ pub(crate) struct PeerConnections {
     pub(crate) sender: RpcSender,
     /// Subscribed topics.
     pub(crate) topics: BTreeSet<TopicHash>,
-    /// Don't send messages.
-    pub(crate) dont_send: LinkedHashMap<MessageId, Instant>,
+    /// Don't send messages received from the peer.
```

Comment:
nit:
```suggestion
    /// IDONTWANT messages received from the peer.
```
since we are modifying this, probably makes sense to make this more explicit, as there aren't don't send messages. Wdyt?